modules/steam.nix

```diff
@@ -57,9 +57,7 @@ in
       systemd.user.services."steam" = {
         enable = true;
         serviceConfig = {
-          KillMode = "process";
           Restart = "always";
-          RestartSec = "1";
```

Comment:
Those weren't needed... I had copied them over from another service I was using for something else.

The KillMode, especially, is problematic here. You don't want anything started by the steam service to linger around when stopped.